test/integration/src/test/java/io/pravega/test/integration/ControllerMetricsTest.java

```diff
@@ -162,7 +162,7 @@ public void streamMetricsTest() {
             // Check that the number of streams in metrics has been incremented.
             streamManager.createStream(scope, iterationStreamName, streamConfiguration);
             Counter createdStreamsCounter = MetricRegistryUtils.getCounter(getCounterMetricName(CREATE_STREAM));
-            Assert.assertTrue(i <= createdStreamsCounter.getCount());
+            Assert.assertTrue(i <= createdStreamsCounter.count());
```

Comment:
You can use `AssertExtensions.assertGreaterThanOrEqualTo`. In that case, your failure message will also include both values.

Reply:
Fixed